src/api_stackflow.js

```diff
@@ -0,0 +1,31 @@
+//install npm i node-fetch --save
```

Comment:
could you run `npm i node-fetch` and push the updated `package.json`? we'll all be able to install the new dependency automatically with `npm i` once that file is updated